pharaoh_examples/test/api_service_test.dart

```diff
@@ -10,6 +10,26 @@ void main() async {
 
     tearDownAll(() => apisvc.app.shutdown());
 
+    group('TEMP: Additional request methods (Change this if PR accepted)', () {
+      test('when using `HEAD` request method', () async {
+        final serverUrl = apisvc.app.uri.toString();
+        final path = Uri.parse('$serverUrl/api/users?api-key=foo');
+
+        final result = await http.head(path);
+        expect(result.statusCode, 200);
+        expect(result.body, '');
+      });
+
+      test('when using `PATCH` request method', () async {
+        final serverUrl = apisvc.app.uri.toString();
+        final path = Uri.parse('$serverUrl/api/user/jane/repos?api-key=foo');
+
+        final result = await http.patch(path);
+        expect(result.statusCode, 200);
+        expect(result.body, 'Updated jane repos');
+      });
+    });
```

Comment:
This shouldn't be here.

Reply:
Moving this to a new file `pharaoh_examples/test/api_method_test.dart`

Reply:
this isn't useful without [this](https://github.com/codekeyz/pharaoh/pull/45/files/c8f0cc0daf36b924dfe062e9823c5a93fd854934#diff-476633b5ed9ab7c51840c4ae6349c833d39fa57ff41b25565fe165b0499667f3) so dropped it